omod/src/main/compass/sass/runReport.scss

```diff
@@ -29,4 +29,8 @@
 img.small {
   width: 24px;
   height: 24px;
+}
+
```

Comment:
you can remove this space too

Reply:
@HerbertYiga I use the space to keep consistence with the remaining file
Should I remove it even then?

Reply:
yes at line 33

Reply:
@HerbertYiga Should be done now